client/src/views/Environment.tsx

```diff
@@ -34,36 +40,55 @@ function Environment() {
     setEndpoints(list);
   }, ENDPOINTS_POLLING_INTERVAL);
 
+
+    useInterval( async () => {
+      if (!environment || !currentContext) return;
+      const status = await okteto.status(environment.file, currentContext.name);
+       if (previousStatus && previousStatus === 'activating' && status === 'synchronizing' )
+         window.ddClient.desktopUI.toast.success('env ready to use');
+        if (!previousStatus || previousStatus !== status)
+          setPreviousStatus(status)
+        setStatus(status);
+    }, STATUS_POLLING_INTERVAL,true);
```

Comment:
Just a minor optimization: to avoid calling this function we can cancel the interval by setting null if the environment is not ready (https://github.com/Hermanya/use-interval#usage):

```suggestion
    }, environment ? STATUS_POLLING_INTERVAL : null, true);
```